source/Becasso.cpp

```diff
@@ -955,7 +955,8 @@ void Becasso::MessageReceived (BMessage *message)
 		}
 		case 'aURL':
 		{
-			char *argv[1] = { "http://www.sumware.demon.nl" };
+			//char *argv[1] = { "http://www.sumware.demon.nl" }; not available anymore
```

Comment:
it's not needed to keep all the old code commented out. That's what the git history is meant for :)

Reply:
Ok, wasn't sure about it. It's my first pr. So next time I will consider. Thanks :-)